### ISSUE TITLE
Release 0.3 to pypi

### ISSUE THREAD

**Issue by diegoponciano** [APSL/wordpress-to-puput#7]
I can't import a XML from my blog with 0.2, but I verified it works installing from master (0.3).  
Can you release the latest to pypi?

**Comment by marctc** [comment]
wordpress-to-puput is currently under the release 0.3 in pypi: https://pypi.python.org/pypi/wordpress-to-puput

**Comment by diegoponciano** [comment]
@marctc version 0.3 is the latest on pypi, but it seems there's no package uploaded for this version, I still can't install it.  
maybe it'll take a while to show up?

**Comment by diegoponciano** [comment]
I'm not too familiar with pypi, but I think you have to run `python setup.py sdist upload -r pypi` to upload the tar.gz package.

**Comment by FlipperPA** [comment]
Howdy @diegoponciano - in the meantime, you should be able to install it like this:

`pip install git+https://github.com/APSL/wordpress-to-puput.git#5100ce4f8cba7fda6476411938e845ea89984bfe`

**Comment by marctc** [comment]
@diegoponciano can you try again please?

**Comment by diegoponciano** [comment]
@marctc Just tried with `pip install wordpress-to-puput==0.3`, didn't work and got this message:  
```
Could not find a version that satisfies the requirement wordpress-to-puput==0.3 (from versions: 0.1, 0.1.1, 0.2)
No matching distribution found for wordpress-to-puput==0.3
```  
Comparing this page https://pypi.python.org/pypi/wordpress-to-puput/0.2 with this one https://pypi.python.org/pypi/wordpress-to-puput/0.3 we can see in 0.3 the tar.gz file isn't available yet.  
I end up installing from github releases, with: `pip install https://github.com/APSL/wordpress-to-puput/archive/0.3.tar.gz`

**Comment by marctc** [comment]
Released new version: 0.3.1
